src/domain/moit/TimeSelectBottomSheet.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, useState } from 'react';
+import styled from '@emotion/styled';
+import { SelectScrollerOption } from '@components/SelectScroller/SelectScroller.option';
+import { palette } from '@styles/theme';
+import { ModalProps } from 'hooks/useModal';
+import { generateArray } from 'utils/generateArray';
+import BottomSheet from '@components/BottomSheet';
+import { SelectScroller } from '@components/SelectScroller';
+import TimeZone from './components/TimeZone';
+import { useSelectTime } from './hooks/useSelectTime';
+
+interface Props {
+  modalProps: ModalProps;
+}
+
+// TODO: 타입 변환시 해당 인덱스로 스크롤 로직 추가
+// TODO: 인풋 로직 개발되면 저장버튼에 싱크 로직 추가
+
+const TimeSelectBottomSheet: FC<Props> = ({ modalProps }) => {
+  const [currentCursor, setCurrentCursor] = useState<TimeZoneCursor>('start');
+  const { hour, min, startTime, endTime } = useSelectTime(currentCursor);
+
+  return (
+    <BottomSheet
+      modalProps={modalProps}
+      dimColor={palette.modal_dim}
+      headerTitle="시간선택"
```

Comment:
P1
```suggestion
      headerTitle="시간 선택"
```
아주 마이너하지만 넘어갈 수 없는 띄어쓰기 수정 ㅎ

Reply:
ㅋㅋㅋㅋㅋㅋ 감사하고 죄송합니다 ...